modules/ppcp-wc-subscriptions/src/RenewalHandler.php

```diff
@@ -406,7 +406,7 @@ private function process_order( \WC_Order $wc_order ): void {
 				return;
 			}
 
-			if ( $wc_order->get_payment_method() === PayPalGateway::ID ) {
+			if ( $wc_order->get_payment_method() === PayPalGateway::ID || $wc_order->get_payment_method() === 'ppec_paypal' ) {
```

Comment:
We call the `get_payment_method()` 6 times inside the same `process_order` method, so does it make sense to store it inside variable on line 259 for example `$payment_method = $wc_order->get_payment_method();` and then use the `$payment_method` variable in all these `if` checks?